### DISCUSSION TITLE
Webservice - filtering

### DISCUSSION THREAD

**Discussion by pjdevries** [joomla/joomla-cms#34100] [Q&A] [answered]
When requesting a list of articles through `v1/content/article(s)`, apparently `Joomla\Component\Content\Administrator\Model\ArticlesModel::populateState()` is not executed. How does one filter the results, to request articles from a specific category for instance?

**Comment by joomdonation** [comment, marked as answer]
The model states are set manually in the controller, see https://github.com/joomla/joomla-cms/blob/4.0-dev/api/components/com_content/src/Controller/ArticlesController.php#L48-L79

**Reply by pjdevries** [reply]
Thanx for that. Apparently I did not search very good.
